app/services/ucas_matching/process_matching_data.rb

```diff
@@ -117,8 +117,18 @@ def process_csv_file(file)
       existing_matches_string = "#{existing_matches} #{'match'.pluralize(existing_matches)}"
 
       message = ":dfe: :ucas: Hello, this is the Apply/UCAS matching robot. I’ve just received a new file from UCAS. It contained #{new_matches_string}, #{updated_matches_string}, and #{existing_matches_string} we’ve already seen."
-      url = Rails.application.routes.url_helpers.support_interface_ucas_matches_url
+      url = Rails.application.routes.url_helpers.support_interface_ucas_matches_url(years: RecruitmentCycle.current_year)
       SlackNotificationWorker.perform_async(message, url)
+
+      send_action_needed_slack_notification
+    end
+
+    def send_action_needed_slack_notification
+      action_needed_matches ||= UCASMatch.select(&:action_needed?).count
+      action_needed_matches_string = "We have #{action_needed_matches} #{'match'.pluralize(action_needed_matches)} requiring action."
```

Comment:
Afterthought: Shall we put an emoji on the front of this? :)

Reply:
Haha. Definitely! Added the classic `:dfe: :ucas:` but suggestions are welcome.